### PR TITLE
v0.6.0 release notes

### PR DESCRIPTION
**Pull request by akaroml**
Please note that the links are relative for easier review. Will be changed to github links as a separate PR.

Please review and sign off.

### CODE REVIEW

File: release-notes/v0.6.0.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+Maven explorer now allows you to switch between flat view and hierarchical view, to your preference.
+
+## Debugging
+
+### Easy Launching for Multi-Main-Class Projects
+
+When you have multiple main classes in your workspace, you can have a special launch configuration to launch whatever is in the active editor. This can be handy when you use hot keys. And here's the new configuration:
+
+```json
```
Comment (Eskibear) [inline release-notes/v0.6.0.md]: jsonc

File: release-notes/v0.6.0.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+## Testing
+
+### Test Configuration Reorganized
+
+Test configurations are very useful in special test setups. Those configurations were originally stored in `launch.test.json`, which generated lots of confusions according to the user feedback.
+
+We listened, and as a result, we **deprecated** `launch.test.json`, and replaced it with regular VS Code settings. Now, the test configurations stay in `settings.json`, which can be global or at the workspace level. And they look like this:
+
+```json
```
Comment (Eskibear) [inline release-notes/v0.6.0.md]: jsonc

File: release-notes/v0.6.0.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+]
+```
+
+For more details, visit [Test Runner Configurations](https://github.com/Microsoft/vscode-java-test/blob/master/runner-config.md).
+
+### Navigate to Source from Test Explorer
+
+Go to the definition of your test cases by clicking on the entries in Test Explorer.
+
+![Nativate to Source from Test Explorer](v0.6.0/navigate.to.source.gif)
```
Comment (Eskibear) [inline release-notes/v0.6.0.md]: Navigate

File: release-notes/v0.6.0.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+*February 2019*
+
+Welcome to the February 2019 release of Java Extension Pack. This document aims to recap the features, bug fixes around Java language support in between releases. Here are some of the key highlights:
+
+- **[IntelliCode](#intellicode)** for faster coding assisted by AI
+- **[Standalone Java Files](#standalone-java-files-supported)** better supported
+- **[Performance](#performance)** improved
+- **[Dependency Auto-Completion](#dependency-auto-completion-for-pom.xml-files)** for editing `pom.xml` files
+
+The release notes are arranged in the following sections releated to VS Code Java focus areas. Here are some further updates:
```
Comment (Eskibear) [inline release-notes/v0.6.0.md]: sections **releated** to

File: release-notes/v0.6.0.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+
+### Multiple Source Folder Supported
+
+What if you have multiple sub-folders that have source code inside and want to go freestyle? We've got you covered too. Just add these folders to source path, then all the code inside will be correctly compiled.
+
+![Multiple Source Folder Support](v0.6.0/multiple.source.folder.gif)
+
+### Hide Temporary Files
+
+After opening a project folder, some extra files are generated inside the folder. These are the temporary files generated by Java Language Server, which relies on the existance of those files to work properly.
```
Comment (Eskibear) [inline release-notes/v0.6.0.md]: the **existence** of

File: release-notes/v0.6.0.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+
+Now it's your choice to hide those files in VS Code. When opening a project folder, Java Language Server asks how you want handle those files. You can hide them globally or just within the current workspace, or leave them as-is.
+
+![Hide Temporary Files](v0.6.0/hide.temp.files.png)
+
+## Performance
+
+### Parallel Build
+
+Thank to the improvement made from upstream JDT project, we can now enalbe parallel build in Java Language Server. By doing so, the time of loading a project can be reduced. The build process is per project. You will get the most performance gain when you have multiple child projects in your workspace.
```
Comment (Eskibear) [inline release-notes/v0.6.0.md]: now **enable** parallel

File: release-notes/v0.6.0.md [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+    },
+    {
+        // Other configuration entry...
+    }
+]
+```
+
+For more details, visit [Test Runner Configurations](https://github.com/Microsoft/vscode-java-test/blob/master/runner-config.md).
+
+### Navigate to Source from Test Explorer
```
Comment (jdneo) [inline release-notes/v0.6.0.md]: My bad. It's not `from Test Explorer` but `from Test Report` 😰

I can provide a gif for you if you need.

File: release-notes/v0.6.0.md [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+
+![IntelliCode](v0.5.0/intellicode.gif)
+
+**Note:** TypeScript/JavaScript, Python, C++, C# and XAML are also supported.
+
+### Generate Getters & Setters
+
+Now you can generate getters and setters for new member variables.
+
+![Generate Getters & Setters](v0.6.0/gettersettergen.gif)
```
Comment (testforstephen) [inline release-notes/v0.6.0.md]: The gif you provide is a quick fix. Actually we also want to mention the bulk generate via context menu "Source Action-> Generate Getters and Setters".

image
image

File: release-notes/v0.6.0.md [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+
+In the sample below, some of the completion lists have items marked with ★. These items are the recommendations provided by IntelliCode according to the current code context.
+
+![IntelliCode](v0.5.0/intellicode.gif)
+
+**Note:** TypeScript/JavaScript, Python, C++, C# and XAML are also supported.
+
+### Generate Getters & Setters
+
+Now you can generate getters and setters for new member variables.
```
Comment (testforstephen) [inline release-notes/v0.6.0.md]: Now you can bulk generate getters and setters for all new member variables.

File: release-notes/v0.6.0.md [hunk 10]
```diff
@@ -0,0 +1,153 @@
+# What's new in Visual Studio Code Java?
+
+*February 2019*
+
+Welcome to the February 2019 release of Java Extension Pack. This document aims to recap the features, bug fixes around Java language support in between releases. Here are some of the key highlights:
+
+- **[IntelliCode](#intellicode)** for faster coding assisted by AI
+- **[Standalone Java Files](#standalone-java-files-supported)** better supported
+- **[Performance](#performance)** improved
+- **[Dependency Auto-Completion](#dependency-auto-completion-for-pom.xml-files)** for editing `pom.xml` files
```
Comment (yaohaizh) [inline release-notes/v0.6.0.md]: Broken link here

File: release-notes/v0.6.0.md [hunk 11]
```diff
@@ -0,0 +1,153 @@
...
+- **[Performance](#performance)**
+- **[Maven](#maven)**
+- **[Debugging](#debugging)**
+- **[Testing](#testing)**
+
+As always, please feel free to [tweet us your feedback](https://twitter.com/intent/tweet?via=code&hashtags=Java%2CHappyCoding) or [open issues](https://github.com/Microsoft/vscode-java-pack/issues).
+
+## Code Editing
+
+## IntelliCode
```
Comment (yaohaizh) [inline release-notes/v0.6.0.md]: Should be `###`

File: release-notes/v0.6.0.md [hunk 12]
```diff
@@ -0,0 +1,153 @@
...
+
+IntelliCode saves you time by putting what you’re most likely to use at the top of your completion list. IntelliCode recommendations are based on thousands of open source projects on GitHub each with over 100 stars. When combined with the context of your code, the completion list is tailored to promote common practices.
+
+IntelliCode works well with popular Java libraries and frameworks like Java SE and Spring. It will help you whether you are doing monolithic web apps or modern microservices.
+
+In the sample below, some of the completion lists have items marked with ★. These items are the recommendations provided by IntelliCode according to the current code context.
+
+![IntelliCode](v0.5.0/intellicode.gif)
+
+**Note:** TypeScript/JavaScript, Python, C++, C# and XAML are also supported.
```
Comment (yaohaizh) [inline release-notes/v0.6.0.md]: Should add link the general intellicode support pages/extension.

File: release-notes/v0.6.0.md (hunk 10 above)
Comment (akaroml) [reply release-notes/v0.6.0.md]: In VS Code, the link works fine. I think VS Code uses a different markdown engine. This release note is optimized for VS Code audience. So I'll leave it as is.

**Pull request merged**